### PR TITLE
[bitnami/zookeeper] feat(zookeeper) added extra volume options to zookeeper chart

### PR DESCRIPTION
**Pull request by fsniper**
**Description of the change**

This change adds extraVolume parameters to the zookeeper chart. 

**Benefits**

one of the benefits is that you can mount keystores to the pods to enable tls communication.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**
To be able to have different keystores per pod, I added $POD_NAME env var that you can use to volumeMount different paths according to your pods name.

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/zookeeper/templates/statefulset.yaml [hunk 1]
```diff
@@ -148,6 +148,8 @@ spec:
             - name: ZOO_PROMETHEUS_METRICS_PORT_NUMBER
               value: {{ .Values.metrics.port | quote }}
             {{- end }}
+            - name: POD_NAME
+              subPathExpr: $(POD_NAME)
```
Comment (javsalgar) [inline bitnami/zookeeper/templates/statefulset.yaml:152]: Shouldn't it be

```
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.name
```

File: bitnami/zookeeper/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 5.9.0
+version: 5.9.1
```
Comment (javsalgar) [inline bitnami/zookeeper/Chart.yaml:3]: Please bump the minor version, it is adding a feature

File: bitnami/zookeeper/templates/statefulset.yaml (hunk 1 above)
Comment (fsniper) [reply bitnami/zookeeper/templates/statefulset.yaml]: You are absolutely right @javsalgar . I copy pasted wrong section from another chart. Thank you.

File: bitnami/zookeeper/Chart.yaml (hunk 2 above)
Comment (fsniper) [reply bitnami/zookeeper/Chart.yaml]: done.

File: bitnami/zookeeper/templates/statefulset.yaml (hunk 1 above)
Comment (fsniper) [reply bitnami/zookeeper/templates/statefulset.yaml]: done.

**Pull request merged**